src/main/java/org/cardanofoundation/explorer/api/repository/ledgersync/DrepInfoRepository.java

```diff
@@ -74,4 +74,12 @@ select max(dri.votingPower) as maxVotingPower , max(dri.activeVoteStake) as maxA
        min(dri.govParticipationRate) as minGovParticipationRate, max(dri.govParticipationRate) as maxGovParticipationRate from DRepInfo dri
           """)
   DRepRangeProjection getDRepRangeValues();
+
+  @Query(
+      """
+    SELECT dri.drepHash as drepHash, dri.activeVoteStake as activeVoteStake
+    FROM DRepInfo dri
+    WHERE dri.drepHash in :dRepHashList
+    """)
+  List<DRepInfoProjection> findByDRepHashIn(@Param("dRepHashList") List<String> drepHashList);
```

Comment:
I will change the function name to _findDRepActiveStakeInHashList_. However, I currently only need to use the _hash_ and _active_vote_stake_ fields in the DRepInfo table. I use _DRepInfoProjection_ here because I want to declare the minimum possible projection to avoid confusion.